erpnext_com/www/docs/user/manual/en/setting-up/users-and-permissions/user-permissions.md

```diff
@@ -15,8 +15,11 @@ When you create a new record you will have to specify
 1. The user for which the rule has to be applied
 2. The type of document which will be allowed (for example "Company")
 3. The specific item that you want to allow (the name of the "Company)
+4. Select "Is Default" to choose the default value that is allowed for a type of document.
 
-<img src="{{docs_base_url}}/assets/img/users-and-permissions/user-perms/new-user-permission.png" class="screenshot" alt="Creating a new user permission">
+    <img src="{{docs_base_url}}/assets/img/users-and-permissions/user-perms/new-user-permission.png" class="screenshot" alt="Creating a new user permission">
```

Comment:
inconsistency in screenshots, show top left navigation part in both screenshots
highlight the Is Default checkbox with a red box

Reply:
It is merely to bring focus on the image, red boxes don't work the way you believe they work. They have special meaning. I don't think this change is required.

Reply:
a red box means that the field is mandatory and adding random red boxes will only add to the confusion. Honestly, adding anything will ruin the visibility of the image, there are barely 4 fields.